rollup.config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  entry: './release/index.js',
+  dest: './release/bundles/router-store.umd.js',
+  format: 'umd',
+  moduleName: 'ngrx.router-store',
```

Comment:
The hyphen here won't work, you will need to camel-case this to `ngrx.routerStore`